README.md

```diff
@@ -52,7 +52,10 @@ giving us both centralized and local logs.  (Redundancy is nice).
 		flag.Parse()
 
 		if graylogAddr != "" {
-			gelfWriter, err := gelf.NewWriter(graylogAddr)
```

Comment:
Why not keeping the NewWrite function which would return the `Writer` interface?
Depending on the string given to the `NewWriter` function, an `UDPWriter` or `TCPWriter` would be returned.
It would be easier to add new writers in the future.